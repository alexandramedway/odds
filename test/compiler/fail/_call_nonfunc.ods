do f = 5
do f("Hello, world!")
