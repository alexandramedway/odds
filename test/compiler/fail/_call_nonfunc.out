[31mSemantic error[0m:
  Attempting to call Num type 'f' as a function
